### PR TITLE
Add Yargs for Cli

### PR DESCRIPTION
**Pull request by AlphaStyle**

### CODE REVIEW

File: bin/cordlr [hunk 1]
```diff
@@ -1,34 +1,50 @@
...
+          describe: 'Prefix to trigger commands',
+          type: 'string'
+        },
+        'token': {
+          alias: 't',
+          describe: 'Discord API Token',
+          type: 'string'
+        },
+        'plugins': {
+          alias: 'p',
```
Comment (bdbch) [inline bin/cordlr]: Do we need this?

File: bin/cordlr [hunk 2]
```diff
@@ -1,34 +1,50 @@
...
+// CLI Data
+require('yargs') // eslint-disable-line
+  .usage('$0 [command, [options, [values]]]')
+  .command({
+    command: 'start',
+    aliases: ['s'],
+    desc: 'Start the bot',
+    builder: (yargs) => {
+      yargs.options({
+        'prefix': {
```
Comment (bdbch) [inline bin/cordlr]: Do we need this?

File: bin/cordlr [hunk 3]
```diff
@@ -1,34 +1,50 @@
...
+          alias: 'x',
+          describe: 'Prefix to trigger commands',
+          type: 'string'
+        },
+        'token': {
+          alias: 't',
```
Comment (bdbch) [inline bin/cordlr]: Do we need this?

**Review by bdbch — CHANGES_REQUESTED** [review]
Do we need these arguments since these options are coming out of the cordlr.json?

**Comment by AlphaStyle** [comment]
Removed Token, Prefix and plugins options to `start` and added `update`, `install`

**Comment by bdbch** [comment]
@AlphaStyle There is now a merge conflict.
could you check that?

**Comment by AlphaStyle** [comment]
A proposal to how the yargs cli can look like. Ideas or feedback ?

**Review by bdbch — APPROVED** [review]

**Pull request merged**